src/main/java/nl/tudelft/semgroup4/GameEndedState.java

```diff
@@ -22,208 +20,406 @@
-                "MONEY", Color.yellow);
-        typeFontPlayer.drawString(container.getWidth() / 5 * 4, container.getHeight() / 4,
-                "NAME", Color.yellow);
-        //draws all information of the fist player
-        typeFontPlayer.drawString(container.getWidth() / 8, container.getHeight() / 3,
-                "PLAYER 1", Color.white);
-        typeFontPlayer.drawString(container.getWidth() / 5 * 2, container.getHeight() / 3,
-                Integer.toString(players[0].getScore()), Color.white);
-        typeFontPlayer.drawString(container.getWidth() / 5 * 3, container.getHeight() / 3,
-                Integer.toString(players[0].getMoney()), Color.white);
-        
-        //draws the first players textfield box.
-        graphics.setColor(Color.white);             
-        textFieldPlayer1.render(container, graphics);
-        
-        //Draws a white rectangle around the textfield because
-        //slick can't handle it on it's own.
-        graphics.drawRect(
-                container.getWidth() / 5 * 4,
-                container.getHeight() / 3,
-                textFieldPlayer1.getWidth() - 1,
-                textFieldPlayer1.getHeight() - 1);
-        
-        //Displays all information of the second player if there is a second player.
-        if (players.length == 2) {
-            typeFontPlayer.drawString(container.getWidth() / 8, container.getHeight() / 2,
-                    "PLAYER 2", Color.white);
-            typeFontPlayer.drawString(container.getWidth() / 5 * 2, container.getHeight() / 2,
-                    Integer.toString(players[1].getScore()), Color.white);
-            typeFontPlayer.drawString(container.getWidth() / 5 * 3, container.getHeight() / 2,
-                    Integer.toString(players[1].getMoney()), Color.white);
-            
-            //Draws the second players textfield box.
-            textFieldPlayer2.render(container, graphics);
-            
-            //Draws a white rectangle around the textfield because
-            //slick can't handle it on it's own.
-            graphics.drawRect(
-                    container.getWidth() / 5 * 4,
-                    container.getHeight() / 2,
-                    textFieldPlayer1.getWidth() - 1,
-                    textFieldPlayer1.getHeight() - 1);
-        }        
-        
-        //draws the continue button on the screen.
-        typeFontPlayer.drawString(
-                container.getWidth() - margin - typeFontPlayer.getWidth(gameContinueText),
+
+    }
+
+    /**
+     * Draws the column names of the player information.
+     * 
+     * @param container
+     *            container to draw onto
+     */
+    public void drawColumnNames(GameContainer container) {
+        // draws the column names
+        typeFontPlayer.drawString(container.getWidth() / 5 * 2,
+                container.getHeight() / 4, "SCORE", Color.yellow);
```

Comment:
You should use a variable to store this 5, so that when you want to change it, you only have to change it in one position instead of 3.